tasks/Functions-admin/manageAllowlist.js

```diff
@@ -43,14 +56,188 @@ async function addOrRemove(action, taskArgs) {
   console.log(`Allowlist updated for oracle ${oracle.address} on ${network.name}`)
 }
 
-task("functions-add-senders", "Add wallets to allowlist in the Oracle contract")
-  .addParam("addresses", "Comma-separated list of addresses")
+task("functions-add-senders", "Add wallets to allowlist in the Oracle contract.  In order to add users from allowlist.csv, copy the CSV file into the root directory and do not set the addresses parameter.")
+  .addOptionalParam("addresses", "Comma-separated list of addresses.  If this is not provided, addresses will be pulled from allowlist.csv")
+  .addOptionalParam("eventcodes", "Comma-separated list of valid event code that must be provided by the user to be added")
```

Comment:
Confusing description: "OptionalParam" vs "must be provided".

Reply:
I have now made eventcodes optional.  If this param is not provided, we will not filter by event code before adding to the allowlist.